src/main.js

```diff
@@ -6,7 +8,20 @@ import "./registerServiceWorker";
 import vuetify from "./plugins/vuetify";
 import "./style.css";
 import firebase from "@/firebase";
-import "material-design-icons-iconfont/dist/material-design-icons.css";
+//import "material-design-icons-iconfont/dist/material-design-icons.css";
+
+
+import cssVars from "css-vars-ponyfill";
+//import "material-design-icons-iconfont/dist/material-design-icons.css";	//import "material-design-icons-iconfont/dist/material-design-icons.css";
+
+
+cssVars({
+  watch: true,
+});
+
+//Vue.use(vueDebounce);	Vue.use(vueDebounce);
```

Comment:
this commented out code can be removed entirely

Reply:
Removed.